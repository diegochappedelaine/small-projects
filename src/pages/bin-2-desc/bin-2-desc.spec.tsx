import { render } from '@testing-library/react';
import Bin2DescPage from './index';

describe('Bin2DescPage', () => {
  it('should render correctly', () => {
    const { container } = render(<Bin2DescPage />);
    expect(container).toMatchSnapshot();
  });
});
