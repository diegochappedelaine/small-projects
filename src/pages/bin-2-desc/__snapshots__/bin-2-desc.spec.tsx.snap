// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bin2DescPage should render correctly 1`] = `
<div>
  <div
    class="chakra-container css-0"
  >
    <div
      class="chakra-stack css-jampdr"
    >
      <h2
        class="chakra-heading css-164r41r"
      >
        Binary To Descimal
      </h2>
      <h2
        class="chakra-heading css-1n49pl7"
      >
        Convert binary input to decimal
      </h2>
      <p
        class="chakra-text css-6dvxm2"
      >
        Binary
      </p>
      <ol
        class="chakra-stack css-84zodg"
      >
        <li
          class="css-gptsw1"
        >
          -
        </li>
        <li
          class="css-gptsw1"
        >
          -
        </li>
        <li
          class="css-gptsw1"
        >
          -
        </li>
        <li
          class="css-gptsw1"
        >
          -
        </li>
        <li
          class="css-gptsw1"
        >
          -
        </li>
        <li
          class="css-gptsw1"
        >
          -
        </li>
        <li
          class="css-gptsw1"
        >
          -
        </li>
        <li
          class="css-gptsw1"
        >
          -
        </li>
      </ol>
      <hr
        aria-orientation="horizontal"
        class="chakra-divider css-1upb9tn"
      />
      <p
        class="chakra-text css-6dvxm2"
      >
        Decimal
      </p>
      <p
        class="css-1qm7lvz"
      >
        0
      </p>
    </div>
  </div>
</div>
`;
